src/tjenesteomrader/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Layout from 'src/layout';
+import style from './index.module.css';
+import digitalTjenesteBlob from './images/indexBlobs/digitalTjeneste.svg';
+import serviceBlob from './images/indexBlobs/strategi.svg';
+import datadrevet from './images/indexBlobs/datadrevet.svg';
+import kultur from './images/indexBlobs/kultur.svg';
+import Entrance from './components/entrance/entrance';
+
+const Tjenesteomrader = () => {
+  return (
+    <Layout>
+      <div className={style.tjenesteomrader}>
+        <h2>Våre tjenesteområder</h2>
+        <p>
+          Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus
+          volutpat nisi, volutpat diam tincidunt viverra. Sagittis, sapien porta
+          non imperdiet aliquam, eu nec. Enim vel at diam massa tortor blandit
+          et commodo risus. Faucibus lacus metus eget eu eget convallis
+          pharetra.
+        </p>
+        <Entrance
+          blobPath={digitalTjenesteBlob}
+          serviceName={'Digital tjeneste-og produktutvikling'}
+          serviceText={
+            'Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus volutpat nisi, volutpat diam tincidunt viverra.'
+          }
+          buttonColor={'#FFFFFF'}
+          buttonBgColor={'#E61A6B'}
+          urlPath={'digitalTjeneste'}
+        />
+        <Entrance
+          blobPath={serviceBlob}
+          serviceName={'Strategi'}
+          serviceText={
+            'Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus volutpat nisi, volutpat diam tincidunt viverra.'
+          }
+          buttonColor={'#333333'}
+          buttonBgColor={'#03DAC6'}
+          urlPath={'strategi'}
+        />
+        <Entrance
+          blobPath={datadrevet}
+          serviceName={'Datadrevet'}
```

Comment:
```suggestion
          serviceName="Datadrevet"
```

Når det ikke er variabler så er det vanligst praksis å skrive slik. Det er ogsa litt mindre "visuell støy" av syntaks

Reply:
Er det greit å sende det inn som en prop på den måten, finnes det en penere løsning?

Reply:
Ja, tenker det er helt fint. Spesielt dersom du ønsker kun å støtte tekst.